_posts/en/newsletters/2024-05-15-newsletter.md

```diff
@@ -0,0 +1,183 @@
+    For this reason, modern Bitcoin opcodes typically use 32-byte
+    hash digests.  However, security in single-party protocols or
+    well-designed multiparty protocols using 20-byte hashes can be
+    increased to make compromise unlikely in less than about
+    2<sup>160</sup> brute-force operations, allowing those protocols to
+    save about 12 bytes per digest.  One case where that might be useful
+    is in implementations of the [eltoo][topic eltoo] protocol (see
+    [Newsletter #284][news284 eltoo].
+
+  - *Support for additional commitments:* `OP_CTV` only succeeds if it
+    is executed within a transaction that contains inputs and outputs
+    that hash to the same value as a provided hash digest.  One of those
+    outputs could be an `OP_RETURN` output that commits to some data
+    that the script creator wants published to the blockchain, such as
+    data necessary to recover LN channel state from a backup.  However,
+    putting data in the witness field would significantly less expensive.
+    The proposed updated form of `OP_CTV` requires an additional piece
+    of data from the witness stack be included when the inputs and
+    outputs are hashed and checked against the hash digest provided by
+    the script creator.  This ensures that data is published to the
+    blockchain with minimal use of block weight.
+
+  The proposal has not received any discussion on the mailing list as of
+  this writing.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.0-beta.rc2][] is a release candidate for the next major
+  version of this popular LN node.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #29845][] updates several `get*info` RPCs to change the
+  `warnings` field from a string to an array of strings so that several
+  warnings can be returned instead of just one.
+
+- [Core Lightning #7111][] Plugins: Can now opt in to handle `check` command on their commands, for more thorough checking.  FIXME:Gustavojfe to write
+
+- [Libsecp256k1 #1518][] adds a `secp256k1_pubkey_sort` function that
+  sorts a set of public keys into a canonical order.  This is useful for
+  both [MuSig2][topic musig] and [silent payments][topic silent
+  payments], and likely many other protocols involving multiple keys.
```

Comment:
This was already reported on in last week's newsletter as part of the release announcement.

Reply:
Yes, but it wasn't reported in this section (because it hadn't been merged when I began writing last week's newsletter).

On several prior ocassions, we've described PRs after previously mentioning them in the release section, since we compile those different sections at different times.  There's no harm in mentioning things twice and often some benefit when the second mention goes into additional detail.